source/js/components/petition/petition-thank-you.jsx

```diff
@@ -0,0 +1,180 @@
+import { Component } from "react";
+import { ReactGA } from "../../common";
+import DonationModal from "./donation-modal.jsx";
+import copyToClipboard from "../../copy-to-clipboard";
+
+/**
+ * Thank you screen for FormAssembly petitions
+ * This is a simplified version of the original Petition component
+ */
+class PetitionThankYou extends Component {
+  constructor(props) {
+    super(props);
+    this.state = this.getInitialState();
+
+    // Do we have modal data?
+    this.modals = false;
+    if (this.props.modals) {
+      this.modals = this.props.modals;
+      try {
+        this.modals = JSON.parse(this.modals);
+      } catch (e) {
+        this.modals = false;
+        console.error(`Could not parse modal data from petition markup.`);
+      }
+    }
+  }
+
+  getInitialState() {
+    return {
+      showDonationModal: true,
+    };
+  }
+
+  // [FA TODO] set up GA event tracking for form submissions
+  setAnalytics() {
+    // window.dataLayer = window.dataLayer || [];
+    // window.dataLayer.push({
+    //   event: "form_submission",
+    //   form_name: this.props.ctaName,
+    //   form_location: this.props.formLocation,
+    //   form_type: "petition-form",
+    //   form_id: this.props.petitionId,
+    // });
+  }
+
+  shareButtonClicked(event, shareProgressButtonId) {
+    // ReactGA.event({
+    //   category: `petition`,
+    //   action: `share tap`,
+    //   label: `${document.title} - share tap`,
+    // });
+
+    if (shareProgressButtonId) {
+      let shareProgressButton = document.querySelector(
+        `#${shareProgressButtonId} a`
+      );
+
+      if (shareProgressButton) {
+        shareProgressButton.click();
+      }
+    } else {
+      // copying current page URL (without query param or hash) to clipboard
```

Comment:
`petition.jsx` does it by copying the full url of the page (`window.location.href`). I'm eliminating query param and hash here as we don't want people to share a link that goes directly to the thank you screen of a petition page.